simapp/simd/cmd/root.go

```diff
@@ -75,6 +79,43 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+// initConfig helps to override default appConfig template and settings.
+// return; if no custom configuration is required for the application.
+func initConfig() (customAppTemplate string, customAppConfig interface{}) {
```

Comment:
is the `customAppTemplate` needed here? seems that it's overwritten on L100

Reply:
These are return variables

Reply:
duh, sorry lol